### DISCUSSION TITLE
Discussion: Verdaccio should warn if htpasswd is not accessible

### DISCUSSION THREAD

**Discussion by jdmichaud** [verdaccio/verdaccio#2449] [Q&A] [answered]
On Verdaccio 4.0.4, if an htpasswd file is provided in the config file and the file is not accessible (f.e. the path is wrong), there is no error or warning in the logs. User might spend a significant amount of time trying to figure out why authentication does not work because of a typo or a access right issue.

It might be a good idea, from a usability point to view to warn the user, in the logs, that the file is provided is not accessible.

**Comment by juanpicado** [comment, marked as answer]
It is already considered in the next major, not only for `htpasswd` (not implemented yet) but for it is already for the `config.yaml` file, a mode deterministic error log, thanks for the feedback @jdmichaud 

🥺 🙏🏼 upgrade, `4.0.4` is out of support already.

Ref: https://github.com/verdaccio/verdaccio/projects/18
